ideascube/mediacenter/views.py

```diff
@@ -50,6 +59,19 @@ def get_context_data(self, **kwargs):
         self._set_available_kinds(context)
         self._set_available_langs(context)
         self._set_available_tags(context)
+        # We assume that if there is a source, it is a package_id.
+        package_id = context.get('source')
+        if package_id:
+            catalog = catalog_mod.Catalog()
+            try:
+                package = catalog.list_installed([package_id])[0]
```

Comment:
> But, do we also need get_installed_package, get_available_package, get_upgradable_package and get_nothandled_package ?

We currently don't need all of these, only the `get_installed_package` version.

But yeah, that's just a minor cosmetic nit, if we end up finding we need it more then maybe we can add it at that point.